src/ui/templates/insurance/start.njk

```diff
@@ -51,4 +51,20 @@
 
   </form>
 
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-three-quarters-from-desktop">
+        <p class="govuk-body" data-cy="sign-in">
+          <span>{{ CONTENT_STRINGS.SIGN_IN.BODY_1 }}</span> <a class="govuk-link govuk-link--no-visited-state" href="{{ CONTENT_STRINGS.SIGN_IN.LINK.HREF }}" data-cy="sign-in-link">{{ CONTENT_STRINGS.SIGN_IN.LINK.TEXT }}</a> <span>{{ CONTENT_STRINGS.SIGN_IN.BODY_2 }}</span>
+        </p>
```

Comment:
Can we have each tag on its own line?

Reply:
it fails the e2e test because it expects a \n then

Reply:
Up to you, in other places we data `data-cy` on each element inside a paragraph and test each element individually.

It does complicates the test; The trade off to not doing this is the nunjucks template is hard to follow.